plugin/wavesurfer.spectrogram.js

```diff
@@ -73,10 +73,29 @@ WaveSurfer.Spectrogram = {
             this.container.removeChild(prevSpectrogram);
         }
 
+        var specLabels = (this.params.labels||false);
```

Comment:
You don't need to do this kind of conversion explicitly. Just check directly `if (this.params.labels)`.